### PR TITLE
Show license information

### PR DESCRIPTION
**Pull request by matthiasclasen**
Now that we have infrastructure for appdata, lets add this field to it before 1.2, so we don't have to bump the version number right away.

### CODE REVIEW

File: app/flatpak-builtins-info.c [hunk 1]
```diff
@@ -224,6 +226,7 @@ flatpak_builtin_info (int argc, char **argv, GCancellable *cancellable, GError *
       len = MAX (len, strlen (_("Arch:")));
       len = MAX (len, strlen (_("Branch:")));
       len = MAX (len, strlen (_("Version:")));
+      len = MAX (len, strlen (_("License:")));
```
Comment (mwleeds) [inline app/flatpak-builtins-info.c]: Seems we're missing an `if (appdata_license != NULL)` here (and similarly for appdata_version but perhaps that should be a separate commit)

File: common/flatpak-installed-ref.c [hunk 2]
```diff
@@ -52,6 +52,7 @@ struct _FlatpakInstalledRefPrivate
   char    *appdata_name;
   char    *appdata_summary;
   char    *appdata_version;
+  char    *appdata_license;
```
Comment (mwleeds) [inline common/flatpak-installed-ref.c]: This needs a `g_free` in the `_finalize()` (and similarly for some of the other properties)

**Comment by matthiasclasen** [comment]
@rh-atomic-bot r=mwleeds
